src/Entity/ThematicCommunity/ThematicCommunity.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Entity\ThematicCommunity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use ApiPlatform\Core\Annotation\ApiProperty;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class ThematicCommunity
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\GeneratedValue
+     *
+     * @ApiProperty(identifier=false)
+     */
+    private $id;
```

Comment:
could you use `IdentityTrait` instead, I think we need to have `uuid` too. 
+ `__construct(...)` for `$this->uuid = ...` initialisation